string mission_id
---
float64 mission_earliest_start